Mathlib/GroupTheory/OrderOfElement.lean

```diff
@@ -211,8 +214,8 @@ theorem orderOf_eq_iff {n} (h : 0 < n) :
     exact h1 ⟨n, h, h'⟩
 
 /-- A group element has finite order iff its order is positive. -/
-@[to_additive
-  /-- A group element has finite additive order iff its order is positive. -/]
+@[to_additive (attr := simp)
```

Comment:
Similar comment here: having a disappearing `orderOf` does not seem unequivocally desirable.